internal/bot/bot.go

```diff
@@ -295,20 +295,30 @@ func (b *Bot) cmdStart(m *dg.MessageCreate) {
 		return
 	}
 
-	err = b.recommendChannel(m.ChannelID)
+	err = b.recommendChannel(m.ChannelID, m.Author.ID)
```

Comment:
recommendChannelはチャンネルを提案するメッセージの作成だけの機能にするべきでは？
```suggestion
	err = b.recommendChannel(m.ChannelID, vchID)
```
"最も参加人数が多い"や"ownerが参加している"vcの検索は別の機能。